src/functionPageUrl.ts

```diff
@@ -18,4 +18,4 @@
 
 const slugify = require('slugify');
 
-exports.functionPageUrl = (name, type) => (`/reference/pxl/${slugify(type.toLowerCase())}/${slugify(name.toLowerCase())}`);
+exports.functionPageUrl = (name, type, basepath) => (`${basepath}/${slugify(type.toLowerCase())}/${slugify(name.toLowerCase())}`);
```

Comment:
annoying nit: can you change the order of arguments. The output is basepath, name, type but the input is backwards.